src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2935,7 +2935,7 @@ void processContextElement
   std::map<std::string, std::string>&  uriParams,   // FIXME P7: we need this to implement "restriction-based" filters
   const std::string&                   xauthToken,
   const std::string&                   apiVersion,
-  bool                                 checkEntityExistance
+  const std::string&                   ngsiv2Flavour
```

Comment:
Actually, there is a RequestType, created in the beginning of the project that I think does just this.
Common code that needs to behave slightly differently depending on the Type of Request